src/tdlib/message.rs

```diff
@@ -119,6 +120,7 @@ mod imp {
                 "id" => obj.id().to_value(),
                 "sender" => obj.sender().to_value(),
                 "is-outgoing" => obj.is_outgoing().to_value(),
+                "can-be-edited" => obj.can_be_edited().to_value(),
```

Comment:
You forgot to actually create the property, you have to add a `ParamSpecBoolean` in the properties vector above.